### PR TITLE
STAT-14689/added new photos according to updated configurations for Zendesk

### PR DESCRIPTION
**Pull request by SnigdhaAm-Kore**
[XOCCAI-STAT-14689-Developer.Review.Checklist.xlsx](https://github.com/user-attachments/files/16053017/XOCCAI-STAT-14689-Developer.Review.Checklist.xlsx)

### CODE REVIEW

**Review by sumitBi-kore — DISMISSED** [review]
Looks good.

**Review by pavanko-Kore — APPROVED** [review]
Looks good.

File: Zendesk/How to install AgentAssist in Zendesk/installation.jpeg
Comment (sumitBi-kore) [inline Zendesk/How to install AgentAssist in Zendesk/installation.jpeg:1]: Just change the sequence of the input fileds, bot id should come before client id.
@SnigdhaAm-Kore

**Review by sumitBi-kore — APPROVED** [review]
Looks good now.

File: Zendesk/How to install AgentAssist in Zendesk/installation.jpeg
Comment (SnigdhaAm-Kore) [reply Zendesk/How to install AgentAssist in Zendesk/installation.jpeg:1]: Done the changes.

**Review by pavanko-Kore — COMMENTED** [review]
Is there any reason, why UXO screens attached here for web/mobile channel details?

**Comment by SnigdhaAm-Kore** [comment]
> Is there any reason, why UXO screens attached here for web/mobile channel details?

In Zendesk, from now we are targeting to support UXO AgentAI V3. It was suggested by Guru. Hence, I updated the screens accordingly.

**Review by pavanko-Kore — APPROVED** [review]
Looks good.

**Review by sumitBi-kore — DISMISSED** [review]

**Review by pavanko-Kore — APPROVED** [review]
Approved

**Review by sumitBi-kore — APPROVED** [review]

**Review by pavanko-Kore — APPROVED** [review]
Looks good.

**Pull request merged**